.github/workflows/update-helm-charts-index.yml

```diff
@@ -0,0 +1,67 @@
+name: hashicorp/waypoint-helm/update-helm-charts-index
+on:
+  push:
+    tags:
+      - 'v[0-9]+.[0-9]+.[0-9]+*'
+jobs:
+  update-helm-charts-index:
+    outputs:
+      go-version: ${{ steps.get-go-version.outputs.version }}
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@ac593985615ec2ede58e132d2e21d2b1cbd6127c # v3.3.0
+      - name: Determine Go version
+        id: get-go-version
+        run: |-
+          echo "Building with Go $(cat ".go-version")"
+          echo "go-version=$(cat ".go-version")" >> "$GITHUB_OUTPUT"
+      - name: Setup Go
+        uses: actions/setup-go@6edd4406fa81c3da01a34fa6f6343087c207a568 # v3.5.0
+        with:
+          go-version: ${{ steps.get-go-version.outputs.go-version }}
+      - name: verify Chart version matches tag version
+        run: |-
+          GO111MODULE=on go install github.com/mikefarah/yq/v2@latest
```

Comment:
`yq` should be available in the ubuntu-latest image already. Can you try removing this to see if it's still available?